crates/op-rbuilder/src/launcher.rs

```diff
@@ -89,17 +78,20 @@ where
     }
 }
 
-impl<B> Launcher<OpChainSpecParser, OpRbuilderArgs> for BuilderLauncher<B>
+pub struct NodeContext {
+    _node: Box<dyn Any>,
```

Comment:
This is done because if the node reference is dropped the node stops + the trait for this reference is pretty complex to write.

Reply:
I never heard that before :) Open to alternatives if you feel that is better